package.json

```diff
@@ -106,7 +106,8 @@
     "// - INTEGRATION": "Integration builds",
     "integration:ng6": "cd integrations/hello-world-ng6 && yarn test",
     "integration:ng8": "cd integrations/hello-world-ng8 && yarn test",
-    "integration:ivy": "cd integrations/ivy && yarn test:ivy"
+    "integration:ng9-ivy": "cd integrations/hello-world-ng9-ivy && yarn test:ivy",
```

Comment:
maybe `"integration:ng9:ivy"`

Reply:
agreed, would be best to keep it consistent